version/1/3/0/de/code_of_conduct.md

```diff
@@ -0,0 +1,24 @@
+# Verhaltenskodex für Mitwirkende
+
+Als die Mitwirkenden und die Verantwortlichen dieses Projekts, und in dem Willen, eine offene und einladende Gemeinschaft zu fördern, geloben wir, alle Personen zu respektieren, die zum Projekt duch das Anlegen von Support-Tickets, dem Veröffentlichen von Feature Requests, dem Überarbeiten von Dokumentation, dem Vorschlagen von Pull Requests oder Patches und durch andere Aktivitäten beitragen.
+
+Wir verpflichten uns, die Mitwirkung an diesem Projekt zu einer belästigungsfreien Erfahrung für alle zu machen, unabhängig von Kenntnisstand, Geschlecht, Geschlechtsidentität und -Ausdruck, sexueller Orientierung, Behinderung, äußerlicher Erscheinung, Körpermaßen, Rasse, Ethnizität, Alter, Religion oder Nationalität.
+
+Beispiele für nicht akzeptables Verhalten beinhalten:
+
+* Die Verwendung sexualisierter Sprache oder Symbolik
```

Comment:
As the person responsible for suggesting "Symbolik" in the first place, I had thought @CoralineAda would rather mean 1. instead of 2. Guess I was wrong there!

FWIW, I like @cfbolz alternative suggestion.